api/data/NMBS/vehicleinformation.php

```diff
@@ -160,6 +160,15 @@ private static function getData($html, $lang, $fast, $occupancyArr, $date, $vehi
                     $departureCanceled = true;
                 }
 
+                // Departed
+                $departureImgNode = $node->children[0]->children[0];
```

Comment:
We should check whether `$node->children[0]` and `$node->children[0]->children[0]` exist, otherwise a 500 error would appear